Subtester/ProjectSettings/AndroidResolverDependencies.xml

```diff
@@ -1,14 +1,15 @@
 <dependencies>
   <packages>
     <package>androidx.annotation:annotation:[1.2.0]</package>
-    <package>com.revenuecat.purchases:purchases-hybrid-common:[1.10.0]</package>
+    <package>com.revenuecat.purchases:purchases-hybrid-common:[3.0.0]</package>
```

Comment:
no entirely sure why this hadn't been updated before, I might have just missed committing it